x-pack/plugins/observability_solution/apm/server/lib/helpers/get_document_sources.ts

```diff
@@ -206,7 +207,13 @@ const mapToSources = (sources: DocumentTypeData[], hasAnySourceDocBefore: boolea
     const { documentType, hasDocAfter, hasDocBefore, rollupInterval, allHaveDurationSummary } =
       source;
 
-    const hasDocBeforeOrAfter = hasDocBefore || hasDocAfter;
+    // To return that serviceTransactionMetric docType is available,
+    // data has to exist before and after to avoid missing data generated by < 8.7
```

Comment:
Is "before" and "after" referring to before and after an upgrade or a timerange? If its the timerange, does "after" mean current time range?